grumphp.yml

```diff
@@ -0,0 +1,16 @@
+grumphp:
```

Comment:
Did GrumPHP change its config file structure?

Reply:
Yes, from version 0.19 "grumphp" is used as a root of configuration instead of "parameters" in 0.18. https://github.com/phpro/grumphp/blob/v0.19.0/grumphp.yml.dist

Reply:
Got it, thanks1